src/rpc/net.cpp

```diff
@@ -237,6 +230,12 @@ static RPCHelpMan getpeerinfo()
                 heights.push_back(height);
             }
             obj.pushKV("inflight", heights);
+            if (statestats.m_ping_usec > 0) {
+                obj.pushKV("pingtime", ((double)statestats.m_ping_usec) / 1e6);
+            }
+            if (statestats.m_ping_wait_usec > 0) {
+                obj.pushKV("pingwait", ((double)statestats.m_ping_wait_usec) / 1e6);
```

Comment:
Can you use named casts in lines 234 and 237 while touching this code?

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Res-casts-named

Reply:
I'd rather not change this while moving it. There's another C-style cast from int64_t to double on line 206. Let's fix them all at the same time.